charts/opensrp-web/values.yaml

```diff
@@ -105,41 +105,27 @@ containerEnvironmentVariables:
 
 # The react app environment variables
 reactEnvironmentVariables:
-  REACT_APP_ACTION_UUID_NAMESPACE: "35968df5-f335-44ae-8ae5-25804caa2d86"
```

Comment:
@peterMuriuki are all the removed envs no longer needed?

Reply:
yes, only leaving envs for fhir-web